src/tests/MyProfile.test.js

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import MyProfile from '../component/MyProfile';
+import store from '../Redux/store';
+/* eslint-disable */
```

Comment:
- [ ] For best practices and as required for all Microverse projects, kindly make sure you fix your linters errors and not disable them.